DeepLearningSuite/DeepLearningSuiteLib/python_modules/tensorflow_detect.py

```diff
@@ -9,7 +9,7 @@
 import tensorflow as tf
 import time
 
-np.set_printoptions(threshold='nan')
+np.set_printoptions(threshold=sys.maxsize)
```

Comment:
I didn't understand why this was necessary ? Could you please explain that ?

Reply:
Hi @vinay0410 

I get a problem with this line for this reason I  have changed it.
Also there was a line without tabulation